tests/FakeItEasy.Tests/Creation/CastleDynamicProxy/CastleDynamicProxyGeneratorTests.cs

```diff
@@ -207,7 +212,11 @@ public void Should_specify_that_no_default_constructor_was_found()
             result.ReasonForFailure.Should().StartWith("No usable default constructor was found on the type");
         }
 
+#if !NETCORE
         [Fact]
+#else
+        [Fact(Skip = "System.AppDomainInitializerInfo is not available in .NET Core. Furthermore, even if we could found another non-static internal class in mscorlib that is type-forwarded to Private.CoreLib on .NET Core it may disappear in the future too.")]

```

Comment:
> Alternatively, we can always make our own private class and use it. Then we know that it won't go away.

Totally agree. Does this test need a type from an external assembly? if yes I think there's the complexity of introducing another test assembly to consider.